### PR TITLE
Add eLife assessment with circle separator

### PR DESCRIPTION
**Pull request by viktoria-mahmud**

### CODE REVIEW

**Review by smoqadam — APPROVED** [review]

File: assets/sass/_variables.scss [hunk 1]
```diff
@@ -4,7 +4,9 @@
 
 // Colors
 $color-primary: #087acc;
+$color-primary-blue: #0b70b9;
```
Comment (discodavey) [inline assets/sass/_variables.scss:7]: Please change name of the variable so it's agnostic.

File: assets/sass/_variables.scss [hunk 2]
```diff
@@ -4,7 +4,9 @@
...
 $color-primary-dark: #0769b0;
+$color-primary-darker-grey: #6d6d6d;
```
Comment (discodavey) [inline assets/sass/_variables.scss:9]: Please change name of the variable so it's agnostic.

File: assets/sass/patterns/molecules/article-section.scss [hunk 3]
```diff
@@ -240,7 +241,11 @@ $arrow-size: 5;
 
       .article-section__related_link {
         border-bottom: none;
-        color: $color-primary;
+        color: $color-primary-darker-grey;
```
Comment (discodavey) [inline assets/sass/patterns/molecules/article-section.scss:244]: See comment on variables

File: assets/sass/patterns/molecules/article-section.scss [hunk 4]
```diff
@@ -240,7 +241,11 @@ $arrow-size: 5;
...
+
+        &:hover {
+          color: $color-primary-blue;
```
Comment (discodavey) [inline assets/sass/patterns/molecules/article-section.scss:247]: See comment on variables

**Review by discodavey — COMMENTED** [review]
Not seeing the example in the Pattern Library locally but from the preview link the three links with the circle separator are grey and not the dark blue. The doi colour is not using the darker version.

**Comment by viktoria-mahmud** [comment]
Colors are changed as follow:
DOI - #6d6d6d
Links - #0b70b9 - no hover color

> Not seeing the example in the Pattern Library locally but from the preview link the three links with the circle separator are grey and not the dark blue. The doi colour is not using the darker version.

File: assets/sass/_variables.scss [hunk 5]
```diff
@@ -7,6 +7,9 @@ $color-primary: #087acc;
 $color-primary-dark: #0769b0;
 $color-primary-light: #b3e5fc;
 
+$color-link-blue: #0b70b9;
```
Comment (discodavey) [inline assets/sass/_variables.scss:10]: variable names shoud not have the colour in them (i.e. blue or grey) Please can you make them agnostic

**Review by discodavey — APPROVED** [review]
@viktoria-mahmud Just check with @chris-huggins if there are supposed to be hover states for these links

**Pull request merged**